### ISSUE TITLE
Administration and Profile pages errors

### ISSUE THREAD

**Issue by MaxMapo** [firefly-iii/firefly-iii#4520]
I am running Firefly III version 5.5.0-beta.3

**Description**
activated new FIREFLY_III_LAYOUT=v2 ,  admin page is empty and profile page with 500 - Server Error

**Extra info**
 

**Bonus points**

- I have read the FAQ at https://bit.ly/FF3-FAQ
- I added a screenshot
- I added log files 
- I was able to replicate the issue on the demo site.

**Comment by JC5** [comment]
What's the error?

**Comment by MaxMapo** [comment]
I hope that this log can help
[ff3-apache2handler-2021-03-21.log](https://github.com/firefly-iii/firefly-iii/files/6178437/ff3-apache2handler-2021-03-21.log)

**Comment by JC5** [comment]
What's the output from the /debug page? I have no error on those pages.

**Comment by MaxMapo** [comment]
the output from debug page is in the attached file, I hope it is the right output, otherwise please suggest me where to get the required information.

[Class 'phpseclib_Crypt_RSA' not found (500 Internal Server Error).pdf](https://github.com/firefly-iii/firefly-iii/files/6185270/Class.phpseclib_Crypt_RSA.not.found.500.Internal.Server.Error.pdf)

**Comment by JC5** [comment]
Copy paste the text from the box please. I'm not going to download random PDFs.

**Comment by MaxMapo** [comment]
Debug information generated at 2021-03-23 22:22:47 Europe/Amsterdam for Firefly III version **5.5.0-beta.3**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.5.0-beta.3 |
| Firefly III API | 1.5.0 |
| PHP | 7.4.16 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | af208dfe-ec7d-4411-a02d-c6981f8ceedd |
| Using docker? | false |
| Telemetry | false |
| Layout | v2 |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | notice, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 12 |
| DB drivers | mysql |
| Current driver | mysql |
| DB version | 16 (exp. 16) |
| Login provider | eloquent  |
| Trusted proxies (.env) |  |

| User info | Value |
| --- | --- |
| Session start | 2021-03-01 00:00:00 |
| Session end | 2021-03-31 23:59:59 |
| Session first | 1997-01-01 00:00:00 |
| User ID | 1 |
| User language | en_GB |
| User locale | it_IT |
| Attempt at "it_IT.utf8" | 'it_IT.utf8' |
| Attempt at "it_IT.UTF-8" | 'it_IT.UTF-8' |
| User agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36 |

**Comment by JC5** [comment]
Thanks, this is helpful!

I pushed a fix to the develop branch. A new develop image is also created. I was hoping you and @4e868df3 could try it to see if it works.

**Comment by 4e868df3** [comment]
Still broken for me.

```
/var/www/firefly-iii/app/Support/Http/Controllers/CreateStuff.php

            'active'               => true,
            'account_role'         => 'cashWalletAsset',
            'opening_balance'      => null,
            'opening_balance_date' => null,
            'currency_id'          => $currency->id,
        ];
 
        $repository->store($assetAccount);
 
        return true;
    }
 
    /**
     * Create new RSA keys.
     */
    protected function createOAuthKeys(): void // create stuff
    {
        // switch on PHP version.
        if (7 === PHP_MAJOR_VERSION) {
            $rsa  = new \phpseclib\Crypt\RSA;
            $keys = $rsa->createKey(4096);
        }
        if (8 === PHP_MAJOR_VERSION) {
            $keys = \phpseclib3\Crypt\RSA::createKey(4096);
        }
 
        [$publicKey, $privateKey] = [
            Passport::keyPath('oauth-public.key'),
            Passport::keyPath('oauth-private.key'),
        ];
 
        if (file_exists($publicKey) || file_exists($privateKey)) {
            return;
        }
        // @codeCoverageIgnoreStart
        Log::alert('NO OAuth keys were found. They have been created.');
 
        file_put_contents($publicKey, array_get($keys, 'publickey'));
        file_put_contents($privateKey, array_get($keys, 'privatekey'));
    }

Arguments

    "Class 'phpseclib\Crypt\RSA' not found"

```

**Comment by Styleflix** [comment]
I don't know if this is already fixed in an unpublished version of firefly iii (mine is 5.5.1) but since my error is slightly different I tought I share this:
When I am trying to access the admin page it only says "Hello, I am empty page". I also set the FIREFLY_III_LAYOUT=v2 option.
Addintional info: I use the german translation. Don't know if this is relevant here.

(also this is my first post on github ever - I just signed up for this ^^ - hope I don't mess up the conversation rules/syntax/...)
